src/main/java/com/researchspace/webapp/filter/OpenIdRemoteUserPolicy.java

```diff
@@ -0,0 +1,102 @@
+package com.researchspace.webapp.filter;
+
+import com.researchspace.core.util.CryptoUtils;
+import java.util.Map;
+import java.util.TreeMap;
+import javax.servlet.http.HttpServletRequest;
+import lombok.AccessLevel;
+import lombok.Setter;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.beans.factory.annotation.Value;
+
+/** Translates OpenId claims form httpRequest headers into user information */
+@Slf4j
+@Setter(AccessLevel.PACKAGE)
```

Comment:
would it make sense to be PROTECTED in case someone extends it?

Reply:
ok.